docs/onboard-openshift.md

```diff
@@ -0,0 +1,76 @@
+# Guide to onboard developers into Openshift
+
+A guide for developers and platform engineers to onboard into containers and OpenShift.
+
+## Introduction
+
+Platform Team set up the OpenShift clusters. Clusters are ready to allocate workloads. Developers want to migrate their applications, but there needs to be a current process to onboard them. This document provides a template that can be used to design your onboarding process or use it as it is.
+
+## Summary
+
+### Developer Team Guide
+
+The Developer Team processes to be ready to start their container journey.
+
+1. Developer Foundational Learning
+2. Application Assessment: Apply Twelve-Factor App for Containers.
+
+[A checklist for The Top Twelve-Factor App for Containers]()
+
+The application owner and the team will perform a high-Level application assessment to understand the application maturity level, risks, potential changes, and feasibility.
+
+- [ ] I. Codebase: One codebase tracked in revision control, many deploys
+- [ ] II. Dependencies: Explicitly declare and isolate dependencies
+- [ ] III. Config: Store config in the environment
+- [ ] V. Build, release, run: Strictly separate build and run stages
+- [ ] XI. Logs: Treat logs as event streams
+- [ ] Security and Compliance: Understand security and compliance to be implemented.
+
+3. Request Access to OpenShift
+
+If necessary for your organization, create an automated-repeatable process to gather information about the application and development team.
+
+### Platform Team Checklist
+
+This process describes the Platform Team process to grant access to the developer team and application to OpenShift, considering application and team requirements.
+
+#### Step 1 Project Setup (Platform)
+
+This process creates all the required resources for the development team and applications based on their requirements.
+
+##### Platform Team Checklist
+
+Developers will require a new namespace, access to the namespace, and other requirements to have their applications functional.
+
+This is a recommended list to start creating your platform guide to successfully onboard teams and applications into OpenShift.
+
+- [ ] 1- Project Provisioning per Team
```

Comment:
What do the numbers mean? 

![image](https://user-images.githubusercontent.com/52387464/228310362-ed1f5d1a-cdd0-4b4a-a988-c9d1c248ff80.png)

Reply:
This is to reflect the original document. The content will be updated by the editor.